sdks/go/pkg/beam/coder.go

```diff
@@ -207,7 +207,7 @@ func JSONEnc(in typex.T) ([]byte, error) {
 }
 
 // JSONDec decodes the supplied JSON into an instance of the supplied type.
-func JSONDec(t reflect.Type, in []byte) (T, error) {
+func JSONDec(t reflect.Type, in []byte) (typex.T, error) {
```

Comment:
<!--thread_id:cc_183452242_t; commit:6eb3f3d9a91b8ad5c3cd97182879c9e2435fab18; resolved:1-->
<!--section:context-quote-->
> **herohde** wrote:
> This should just be T.

<!--section:body-->
Done, and changed the other codec functions values to use it as well for consistency.